package.json

```diff
@@ -59,11 +60,17 @@
   "engines": {
     "node": ">=10.15.1"
   },
-  "files": [ "dist" ],
+  "files": [
+    "dist"
+  ],
   "dependencies": {
     "node-fetch": "2.3.0"
   },
   "devDependencies": {
+    "@babel/cli": "7.4.3",
+    "@babel/core": "7.4.3",
+    "babel-preset-latest-node": "3.1.1",
+    "babel-preset-minify": "0.5.0",
     "eslint": "5.16.0",
     "eslint-config-prettier": "4.1.0",
     "eslint-plugin-prettier": "3.0.1",
```

Comment:
Are you planning on supporting Edge? If so, you should add it to your `browserslist`.

Reply:
I was aiming for the browser set that Launch targets. I thought that was just Chrome, Firefox, and Safari? Though, now that you mention it, I realize this SDK is pretty much vanilla JavaScript with no real browser dependencies, so maybe I _should_ include Edge.

And that reminds me that I really need to go back to testing this in all the browsers I claim to support!

Reply:
Launch also supports Edge.